src/rsa/padding.rs

```diff
@@ -63,47 +63,48 @@ impl Encoding for PKCS1 {
 }
 
 impl Verification for PKCS1 {
-    fn verify(&self, msg: untrusted::Input, m: untrusted::Input,
+    fn verify(&self, msg: untrusted::Input, m: &mut untrusted::Reader,
               _mod_bits: usize) -> Result<(), error::Unspecified> {
-        m.read_all(error::Unspecified, |em| {
-            if try!(em.read_byte()) != 0 ||
-               try!(em.read_byte()) != 1 {
-                return Err(error::Unspecified);
-            }
+        let em = m;
 
```

Comment:
:sigh: All of these changes are just supposed to be un-indenting and re-wrapping the code.